packages/core/src/utils/countTrailingZeros.ts

```diff
@@ -11,6 +11,9 @@ export function countTrailingZeros<TAmount>(
   const equalFn = equal(calculator);
 
   return (input: TAmount, base: TAmount) => {
+    if (equalFn(calculator.zero(), input)) {
```

Comment:
Could you add a test for this branch in [this test suite](https://github.com/dinerojs/dinero.js/blob/main/packages/core/src/utils/__tests__/countTrailingZeros.test.ts)?